### ISSUE TITLE
Restructure the files and folders

### ISSUE THREAD

**Issue by ghefraim** [tineret-spearanta-oradea/hopecamp-website#107]
We need to restructure the files and folders. Right now, we don't have a clear structure.

I'm actually thinking of migrating the whole app to NEXT.js (would make it faster and better structured)

This should be done after the camp, since proper testing needs to be done after such a large refactoring.

**Comment by ghefraim** [comment]
I've looked a bit into VueJs. Unfortunately, I feel like it's a bit more different than react. Due to our team's familiarity with React, I think we should continue with Next.Js and shadcn/ui components.

I will probably do this in waves. First, migrate to NextJs, and then gradually change our components to shadcn/ui components. 

What do you think? @DenisWithOneN

**Comment by DenisWithOneN** [comment]
I've also looked a bit into NextJS, it looks promising. As you said since our team is familiar with React I think it's better we go with Next. Though I saw that there are new concepts like server side rendering and such, but I'm sure we'll get familiar with those as well (or at least I need to because right now I didn't encounter something similar). As for shadcn/ui, we'll see how we can implement them in our project, probably using it for generic sections like dashboards or forms, but I think the main design should be custom made. It takes more time but I think we'll have more clarity and freedom on getting it in the direction we want. We can decide more on that after we have a first version of the new design.
